src/AffectedUserFlow/CodeInput/CodeInputForm.tsx

```diff
@@ -243,20 +243,21 @@ const style = StyleSheet.create({
     ...Buttons.primary,
   },
   buttonText: {
-    ...Typography.buttonTextPrimary,
+    ...Typography.buttonPrimaryText,
   },
   disabledButton: {
     ...Buttons.primaryDisabled,
   },
   disabledButtonText: {
-    ...Typography.buttonTextPrimaryDisabled,
+    ...Typography.buttonPrimaryDisabledText,
   },
   secondaryButton: {
     ...Buttons.secondary,
   },
   secondaryButtonText: {
-    ...Typography.buttonTextSecondary,
+    ...Typography.buttonSecondaryText,
   },
 })
 
 export default CodeInputForm
+
```

Comment:
I believe the linting will be sad about this

Reply:
Yeah I always forget to format before throwing up PRs. 🦖